effekt/shared/src/main/scala/effekt/core/LambdaLifting.scala

```diff
@@ -108,7 +108,8 @@ object LambdaLifting extends Phase[CoreTransformed, CoreTransformed] {
   def run(input: CoreTransformed)(using Context): Option[CoreTransformed] =
     input match {
       case CoreTransformed(source, tree, mod, core) =>
-        Some(CoreTransformed(source, tree, mod, lift(core)))
+        val lifted = Context.timed(phaseName, source.name)(lift(core))
```

Comment:
```suggestion
        val lifted = Context.timed(phaseName, source.name) { lift(core) }
```
Same here and in the following cases.